.gitignore

```diff
@@ -52,3 +52,4 @@ lerna-debug.log*
 
 temp.cypher
 temp.graphql
+temp.json
```

Comment:
Is this necessary?

Reply:
well it helps me. I assume it doesn't bother anyone else. I've found that I'll accidently commit temp files and this just keeps it out of the repo.

Reply:
FYI I put those in my global gitignore file so I don't have to keep adding them to each repos ignore.

Reply:
and that is a better idea. I'll change it.